modules/logging-rn-5.6.6.adoc

```diff
@@ -0,0 +1,36 @@
+//module included in logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="logging-release-notes-5-6-6_{context}"]
+= Logging 5.6.6
+This release includes link:https://access.redhat.com/errata/RHBA-2023:3194[OpenShift Logging Bug Fix Release 5.6.6].
+
+[id="logging-5-6-6-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, dropping of messages occurred when configuring the `ClusterLogForwarder` custom resource to write to a Kafka output topic that matched a key in the payload due to an error. With this update, the issue is resolved by prefixing Fluentd's buffer name with an underscore. (link:https://issues.redhat.com/browse/LOG-3458[LOG-3458])
+
+* Before this update, premature closure of watches occurred in fluentd when inodes were reused and there were multiple entries with the same inode. With this update, the issue of premature closure of watches in the fluentd position file is resolved. (link:https://issues.redhat.com/browse/LOG-3629[LOG-3629])
```

Comment:
Fluentd vs fluentd capitalization is inconsistent, I'd pick one and use it consistently in the doc